src/client/scripts/esm/chess/variants/variant.ts

```diff
@@ -33,6 +44,46 @@ const coaIPPromotionsAllowed = repeatPromotionsAllowedForEachColor(coaIPPromotio
 
 const gameruleModificationsOfOmegaShowcasings = { promotionRanks: null, moveRule: null, turnOrder: ['black', 'white'] }; // No promotions, no 50-move rule, and reversed turn order.
 
+interface Moveset {
```

Comment:
Can't help but feel movesets.js would be a better place to store this interface

Reply:
While JsDoc can define types, it apparently can't define types with variable keys (where values are accessed with index notation instead of dot notation).